feebleline.el

```diff
@@ -56,7 +57,19 @@
 (require 'advice)
 (defvar feebleline/mode-line-format-default)
 (defvar feebleline/timer)
-(defvar feebleline/default-log-max)
+(defvar feebleline-mode-line-text
```

Comment:
This new variable is the main thing about the pull request. It makes (I hope) the text in the mode-line easier to customize, and also allows for text properties of different "parts" of the mode-line.